lib/features/children/details/widgets/child_giving_allowance_card.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app/features/children/details/models/profile_ext.dart';
+import 'package:givt_app/features/children/utils/child_date_utils.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/utils/utils.dart';
+
+class ChildGivingAllowanceCard extends StatelessWidget {
+  const ChildGivingAllowanceCard({
+    required this.profileDetails,
+    this.onPressed,
+    super.key,
+  });
+
+  final ProfileExt profileDetails;
+  final void Function()? onPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    final locals = AppLocalizations.of(context);
```

Comment:
You can do `context.l10n`

Reply:
great, that approach looks much more clear, thank you